design/AstroMancer Cards.md

```diff
@@ -0,0 +1,21 @@
+##AstroMancer Cards
+
+###AstroMancer Cards can be played for 2 Mana
+
+Starfall -2 Mana Quadra-cast(Total cost = 2) Deal 6 damage to a random enemy. Feed: Heal a random ally 2.(So 8 healing in all.)
```

Comment:
What does `Feed` mean?

What does `Total cost = 2` mean?

Reply:
So it costs 2 mana to play the card. When resolving the card, it resolves 4 times. 

The "(Total cost = 2)" is to prevent people from thinking that you would have to pay 8 mana to play this card.

Feed means it adds the stated affect to the next card of the cycle.(Stellar feeds into Solar. Solar feeds into Lunar.)